### PR TITLE
paul edit 04

### PR DESCRIPTION
**Pull request by Perevodildo**

### CODE REVIEW

File: Ningen Kakumei (1995) - 04.en.ass [hunk 1]
```diff
@@ -49,9 +49,9 @@ Dialogue: 0,0:32:23.34,0:32:30.60,Sign,,0,0,0,,{\alpha&HF0&\t(30,1000,.5,\alpha&
...
 Dialogue: 0,0:00:40.39,0:00:43.56,Default,,0,0,0,,You seem to be pretty excited, Sakata-kun.
-Dialogue: 0,0:00:43.56,0:00:47.57,Default,,0,0,0,,Yes. You've appointed Iwata-san the dispatched roundtable moderator, {派遣幹部 - basically toda can't attend all those roundtables himself so he trains new leaders to organize discussions for him and sends them out, unsure if moderator is a fitting word}
+Dialogue: 0,0:00:43.56,0:00:47.57,Default,,0,0,0,,Yes. You've appointed Iwata-san to moderate future roundtables, {派遣幹部 - basically toda can't attend all those roundtables himself so he trains new leaders to organize discussions for him and sends them out, unsure if moderator is a fitting word}
 Dialogue: 0,0:00:47.57,0:00:51.32,Default,,0,0,0,,so I decided I also need\Nto rise to his level as soon as possible.
-Dialogue: 0,0:00:51.32,0:00:55.01,Top,,0,0,0,,I see. Good to know our young men department is so full of vim and vigor.
+Dialogue: 0,0:00:51.32,0:00:55.01,Top,,0,0,0,,I see. Good to know the young men of our department are so full of vim and vigor.
```
Comment (Perevodildo) [inline Ningen Kakumei (1995) - 04.en.ass]:
```suggestion
Dialogue: 0,0:00:51.32,0:00:55.01,Top,,0,0,0,,I see. Good to know our young men's division is so full of vim and vigor.
```

Meant to say Division there, as in "Young Men's Division of Soka Gakkai," should this still be changed?

File: Ningen Kakumei (1995) - 04.en.ass [hunk 2]
```diff
@@ -163,27 +163,27 @@ Dialogue: 0,0:07:41.46,0:07:42.44,Default,,0,0,0,,Here it is,
...
-Dialogue: 0,0:07:51.37,0:07:53.99,Narrator,,0,0,0,,in place of "Value Creation" journal,
-Dialogue: 0,0:07:53.99,0:07:59.54,Narrator,,0,0,0,,so a planning meeting was held \Nwith Konishi, Yamadaira and other editors.
+Dialogue: 0,0:07:51.37,0:07:53.99,Narrator,,0,0,0,,in place of the "Value Creation" journal,
+Dialogue: 0,0:07:53.99,0:07:59.54,Narrator,,0,0,0,,so a planning meeting was held \Nwith Konishi, Yamadaira, and other editors.
 Dialogue: 0,0:08:00.56,0:08:02.16,Default,,0,0,0,,That's it, Sensei.
 Dialogue: 0,0:08:02.16,0:08:03.65,Default,,0,0,0,,We should by all means include
 Dialogue: 0,0:08:03.65,0:08:08.18,Default,,0,0,0,,a column for personal experiences and one for refuting the dogmas of other sects.
 Dialogue: 0,0:08:08.18,0:08:09.34,Default,,0,0,0,,Don't you agree?
-Dialogue: 0,0:08:09.87,0:08:13.91,Default,,0,0,0,,Where do you think the false religious views \Nof a modern man come from?
+Dialogue: 0,0:08:09.87,0:08:13.91,Default,,0,0,0,,Where do you think modern men are getting these false religious views from?
```
Comment (Perevodildo) [inline Ningen Kakumei (1995) - 04.en.ass]:
```suggestion
Dialogue: 0,0:08:09.87,0:08:13.91,Default,,0,0,0,,Where do you think young people are getting these false religious views from?
```

it didn't refer to males exclusively

File: Ningen Kakumei (1995) - 04.en.ass [hunk 3]
```diff
@@ -298,24 +298,25 @@ Dialogue: 0,0:18:04.31,0:18:07.32,Default,,0,0,0,,Let's call it a day, then.
...
 Dialogue: 0,0:19:00.95,0:19:05.47,NarrTop,,0,0,0,,and turn it into a bulwark against communism, the US orders the Japanese government
-Dialogue: 0,0:19:05.47,0:19:10.71,NarrTop,,0,0,0,,to enact the "Nine Principles for Economic Stability," also known as Dodge Plan. {https://en.wikipedia.org/wiki/Dodge_Line}
+Dialogue: 0,0:19:05.47,0:19:10.71,NarrTop,,0,0,0,,to enact the "Nine Principles for Economic Stability," also known as the Dodge Plan. {https://en.wikipedia.org/wiki/Dodge_Line}
 Dialogue: 0,0:19:05.46,0:19:10.71,Sign,,0,0,0,,Minister and Economic Adviser to the Supreme Commander for the Allied Powers, Dodge (GHQ) {GHQ連合国総司令部ドッジ公使・経済顧問}
 Dialogue: 0,0:19:10.71,0:19:13.17,Sign,,0,0,0,,Nine principles for economic stability {経済安定九原則}
-Dialogue: 0,0:19:11.21,0:19:14.98,NarrTop,,0,0,0,,It hastened the recovery of Japanese economy's self-reliance.
+Dialogue: 0,0:19:11.21,0:19:11.21,NarrTop,,0,0,0,,It hastened the recovery of self-reliance in Japan'd economy.
```
Comment (Perevodildo) [inline Ningen Kakumei (1995) - 04.en.ass]:
```suggestion
```

what happened there lol

File: Ningen Kakumei (1995) - 04.en.ass [hunk 4]
```diff
@@ -489,22 +490,22 @@ Dialogue: 0,0:30:03.97,0:30:05.99,Default,,0,0,0,,Deciding that is your role. {k
...
-Dialogue: 0,0:30:45.97,0:30:47.60,Default,,0,0,0,,except your own pitiful life.
-Dialogue: 0,0:30:47.60,0:30:55.79,Default,,0,0,0,, Nam-Myoho-Renge-Kyo...
+Dialogue: 0,0:30:25.16,0:30:27.95,Thought,,0,0,0,,Toda-sensei gave me a scolding.
+Dialogue: 0,0:30:27.95,0:30:29.84,Thought,,0,0,0,,It was my own fault.
+Dialogue: 0,0:30:29.84,0:30:32.13,Thought,,0,0,0,,I must reflect on my actions.
+Dialogue: 0,0:30:32.13,0:30:33.81,Thought,,0,0,0,,I have to believe in him
+Dialogue: 0,0:30:33.81,0:30:36.59,Thought,,0,0,0,,and keep working hard, \Nno matter for how long.
+Dialogue: 0,0:30:36.59,0:30:37.69,Thought,,0,0,0,,I must always reflect
+Dialogue: 0,0:30:37.69,0:30:39.56,Thought,,0,0,0,,on my own wrongdoings
+Dialogue: 0,0:30:39.56,0:30:42.39,Thought,,0,0,0,,to become a revolutionary. {unsure, maybe "someone who can start a revolution"}
```
Comment (Perevodildo) [inline Ningen Kakumei (1995) - 04.en.ass]:
```suggestion
Dialogue: 0,0:30:39.56,0:30:42.39,Thought,,0,0,0,,to become a revolutionized person. {unsure, maybe "someone who can start a revolution", but from past lines more likely "someone who underwent revolution"}
```

I still think he means "someone who underwent revolution" (like in the show's title).

File: Ningen Kakumei (1995) - 04.en.ass (hunk 3 above)
Comment (cpuwhiz11) [reply Ningen Kakumei (1995) - 04.en.ass]: I swear I fixed this. Maybe I undid it.

Comment (cpuwhiz11) [reply Ningen Kakumei (1995) - 04.en.ass]: Huh weird why did it double?

**Pull request merged**